src/devices/x10_rf.c

```diff
@@ -69,8 +69,8 @@ static int x10_rf_callback(r_device *decoder, bitbuffer_t *bitbuffer) {
 
     data = data_make(
             "model",    "", DATA_STRING, "X10-RF",
-            "houseid",  "", DATA_STRING, housecode,
-            "deviceid", "", DATA_INT, bDeviceCode + 1,
+            _X("channel", "houseid"),  "", DATA_STRING, housecode,
+            _X("id", "deviceid"), "", DATA_INT, bDeviceCode + 1,
```

Comment:
Can you swap both lines? model, id, channel is the common format.

Reply:
Will do... I wish every change request could be done in three key strokes!